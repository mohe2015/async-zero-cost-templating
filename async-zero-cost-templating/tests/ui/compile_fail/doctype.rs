extern crate alloc;

use async_zero_cost_templating::html;

pub fn main() {
    let (tx, rx) = tokio::sync::mpsc::channel(1);
    let _ = async move {
        html! {
            <!doctype html>
            <html
        }
    };
}
