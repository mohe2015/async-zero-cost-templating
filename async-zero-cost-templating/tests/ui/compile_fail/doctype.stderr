error: unexpected end of input, expected one of: `if`, `for`, identifier, curly braces
  --> tests/ui/compile_fail/doctype.rs:8:9
   |
8  | /         html! {
9  | |             <!doctype html>
10 | |             <html
11 | |         }
   | |_________^
   |
   = note: this error originates in the macro `html` (in Nightly builds, run with -Z macro-backtrace for more info)

error: [note] while parsing attribute
  --> tests/ui/compile_fail/doctype.rs:8:9
   |
8  | /         html! {
9  | |             <!doctype html>
10 | |             <html
11 | |         }
   | |_________^
   |
   = note: this error originates in the macro `html` (in Nightly builds, run with -Z macro-backtrace for more info)

error: [note] while parsing element
  --> tests/ui/compile_fail/doctype.rs:10:13
   |
10 |             <html
   |             ^

error: [note] while parsing child
  --> tests/ui/compile_fail/doctype.rs:10:13
   |
10 |             <html
   |             ^

error: [note] while parsing children
 --> tests/ui/compile_fail/doctype.rs:9:13
  |
9 |             <!doctype html>
  |             ^
